receiver/sqlqueryreceiver/README.md

```diff
@@ -37,8 +37,9 @@ Each _metric_ in the configuration will produce one OTel metric per row returned
 * `aggregation` (optional): only applicable for `data_type=sum`; can be `cumulative` or `delta`; defaults to `cumulative`.
 * `description` (optional): the description applied to the metric.
 * `unit` (optional): the units applied to the metric.
-* `static_attributes` (optional): static attributes applied to the metrics
-
+* `static_attributes` (optional): static attributes applied to the metrics.
+* `start_ts_column` (optional): start timestamp associated with metrics.
+* `stop_ts_column` (optional): stop timestamp associated with metrics, used for setting metric `timestamp`; defaults to time of processing.
```

Comment:
I don’t see where it defaults to time of processing.

Reply:
Thanks for pointing that out! I've edited the Readme accordingly.